src/pages/timecard/Timecard.jsx

```diff
@@ -45,9 +46,14 @@ const Timecard = () => {
   ];
 
   useEffect(() => {
+    let isMounted = true;
```

Comment:
Not entirely sure why this and the logic below is needed?

Reply:
Ah so I was getting this error under a rare circumstance:
`Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in a useEffect cleanup function.`

and it was suggested that the above logic would fix it, but re-testing it doesn't seem to solve much. 
Have removed.